### PR TITLE
docs: add translation flow to README

### PR DESCRIPTION
**Pull request by dferber90**
Documents our translations workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,3 +117,19 @@ By default, only _Deploy Previews_ (Pull Requests) are deployed to [Netlify][net
 [storybook]: https://storybook.js.org
 [netlify]: https://www.netlify.com
 [travis]: https://travis-ci.org/commercetools/ui-kit/
+
+## Translations (i18n)
+
+The UI Kit uses `react-intl`. The core messages are written down in `messages.js` files.
+
+The translations for the supported languages exist in the `/i18n` folder. We do not use any automated translation software.
```
Comment (lufego) [inline README.md]: are there any plans to use it?

File: README.md [hunk 2]
```diff
@@ -117,3 +117,19 @@ By default, only _Deploy Previews_ (Pull Requests) are deployed to [Netlify][net
...
+
+### Adding new messages
+
+In case you are working on a component and you want to add a completely new message, you should add it to (or create a) `messages.js` file. See the other `messages.js` files as an example.
```
Comment (lufego) [inline README.md]: looks good enough. I maybe would have written `as a reference`

File: README.md [hunk 3]
```diff
@@ -117,3 +117,19 @@ By default, only _Deploy Previews_ (Pull Requests) are deployed to [Netlify][net
...
+
+After adding the message, you need to run `yarn i18n:build`. This will modify the language files in `/i18n` by adding the new message.
```
Comment (lufego) [inline README.md]: could be more than one. Maybe `the new message(s).`

File: README.md (hunk 1 above)
Comment (dferber90) [reply README.md]: https://github.com/commercetools/ui-kit/issues/46#issuecomment-421256695

File: README.md (hunk 3 above)
Comment (dferber90) [reply README.md]: Done in a9b2469455f4beba4a3a80cc16f3af571e8c809b

**Pull request merged**